test/api/resources/node.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright © 2017 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+import APIResource from 'api/apiResource';
+import NodeResource from 'api/resources/node';
+
+describe('NodeResource', () => {
+	const defaultBasePath = 'http://localhost:1234';
+	const path = '/node';
+
+	let liskAPI;
+	let resource;
+
+	beforeEach(() => {
+		liskAPI = {
+			headers: {},
+			nodeFullURL: defaultBasePath,
+			hasAvailableNodes: () => {},
+			randomizeNodes: () => {},
+			banActiveNodeAndSelect: () => {},
+		};
+		resource = new NodeResource(liskAPI);
```

Comment:
Needs to be async now. (Also applies to other tests.)

Reply:
Yes! i guess depends on which PR goes in first =0